src/Tools/ExternalAccess/VisualDiagnostics/Contracts/IVisualDiagnosticsLanguageService.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.ServiceHub.Framework;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.Contracts
+{
+    /// <summary>
+    /// Process Information
+    /// </summary>
+    /// <param name="ProcessId">Unique GUID that uniquely identify a process under a debug session</param>
+    /// <param name="LocalProcessId">local process running on a host device, if the process running on a different host (like mobile device), this will be null</param>
+    /// <param name="Path">path to the process, this is guaranteed to not be null for local process, but could be null for mobile devices</param>
+    /// <param name="ConnectionId">Connection Id identifying a data bridge connection to this process</param>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0051:Add internal types and members to the declared API", Justification = "https://github.com/dotnet/roslyn-analyzers/issues/7237")]
+    internal record struct ProcessInfo(Guid ProcessId, uint? LocalProcessId, string? Path, string? ConnectionId);
+
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0051:Add internal types and members to the declared API", Justification = "https://github.com/dotnet/roslyn-analyzers/issues/7237")]
+    internal record struct ConnectionInfo(string? ConnectionId, string? Handle, string? Address, uint? PortNumber);
```

Comment:
I reviewed dotnet/roslyn-analyzers#7237 and found that this is not a bug. These suppressions need to be removed, and the resulting members need to be added to the API files.

Reply:
Since EA.D not longer response to the service broker, the code has been moved inside VisualDiagnosticsLanguageService